edreader/cargo.go

```diff
@@ -72,7 +72,8 @@ func handleCargoFile(file string) {
 }
 
 func renderCargo(page *mfd.Page, cargo Cargo) {
-	page.Add("#  Cargo: %3d  #", cargo.Count)
+	page.Add("#    Cargo     #")
```

Comment:
Could you get it into one line?
```
page.Add("#Cargo: %03d/%03d#")
```
_should_ render as 16 characters if I'm counting right.

That way you have to scroll one line less. But, if you think it looks too cramped that way, then just let me know.

Reply:
I added spaces after "#" character while I was testing and line was scrolling
Actually page.Add("#Cargo: %03d/%03d#") looks good, I will add changes